plugin-quicklaunch/quicklaunchaction.cpp

```diff
@@ -122,8 +164,21 @@ void QuickLaunchAction::execAction(QString additionalAction)
     switch (m_type)
     {
         case ActionLegacy:
-            QProcess::startDetached(exec);
+        {
+            QStringList args;
+#if (QT_VERSION >= QT_VERSION_CHECK(5,15,0))
+            args = QProcess::splitCommand(exec);
+#else
+            args = _splitCommand(exec);
+#endif
+            if (args.isEmpty()) {
+                qWarning() << Q_FUNC_INFO << "Empty argument list";
+            } else {
+                const QString prog = args.takeFirst();
+                QProcess::startDetached(prog, args);
```

Comment:
I would rather use the old good C99 `system(3)`. We don't do any checking or error reporting here anyway.